src/applications/pensions/config/chapters/06-additional-information/documentUpload.js

```diff
@@ -67,7 +67,7 @@ const UploadMessage = (
 
 export default {
   uiSchema: {
-    'ui:title': 'Submit your supporting documents',
+    ...titleUI('Submit your supporting documents'),
     'ui:description': Description,
```

Comment:
move to titleUI arg

Reply:
This description includes an 'h3' that would deviate from the other 'h3' styles on the page. I opted to keep it separate from the 'titleUI' so that the styling would be consistent on this page.